src/__test__/is-printing.spec.ts

```diff
@@ -0,0 +1,30 @@
+import isPrinting from '../is-printing';
+import { SnapshotData, TableLayout, ViewService, ViewState } from '../types';
+
+describe('is-printing', () => {
```

Comment:
I would say you should move the `isPrinting` function + the test to `renderAsPagination`.

Reply:
ah missed that one